### DISCUSSION TITLE
Marlin 2.1.x ?!

### DISCUSSION THREAD

**Discussion by moonglow** [moonglow/FlashForge_Marlin#114] [Q&A]
Marlin has moving forward as it ever be, so new version is already announced 2.1.x ! I think i will able to port it all to new branch... do we need goes forward or waits a little to make it stabilize ?

**Comment by tedkunich** [comment]
I'm game ;)

**Comment by freeifier** [comment]
lets go with the next version and make that good :)

**Reply by moonglow** [reply]
I will finish release of latest 2.0.9.5 and next we will bring new version here !
